src/containers/EstadisticasPage.js

```diff
@@ -1,20 +1,130 @@
-import React, { Component } from 'react';
+import React, { Component, PropTypes } from 'react';
 import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as statisticActions from '../actions/statisticActions';
+import AnimalStatistic from '../components/statistics/AnimalStatistic';
+import AdoptionStatistic from '../components/statistics/AdoptionStatistic';
+import * as valid from '../util/validateForm';
+import { parseAdopterStat, parseAnimalStat } from '../api/statisticApi';
+import '../styles/statistics.scss';
 
 class EstadisticasPage extends Component {
   constructor(props, context) {
     super(props, context);
+
+    this.state = {
+      dates_adoption: {
+        start_date: '',
+        end_date: ''
+      },
+      loadingBar: true,
+      loadingPie: true,
+      errors: {
+        start_date: '',
+        end_date: ''
+      },
+      show_dates_adoption: {
+        start_date: '',
+        end_date: '',
+      }
+    };
+
+    this.onChangeAdoption = this.onChangeAdoption.bind(this);
+    this.onClickRefreshAdoption = this.onClickRefreshAdoption.bind(this);
+  }
+
+  componentWillMount() {
+    this.props.actions.loadDefaultAdoptionStatistic();
+    this.props.actions.loadAnimalStatistic();
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.adoptStat) {
+      this.setState({ loadingBar: false });
+    }
+    if (nextProps.animalStat) {
+      this.setState({ loadingPie: false });
+    }
+  }
+
+  onClickRefreshAdoption() {
+    if (valid.validateDateStatistic(this.state.dates_adoption.start_date, this.state.dates_adoption.end_date)) {
```

Comment:
Largo.
Acostumbrate a usar cosas como `let { start_date, end_date } = this.state.dates_adoption`